ansible/roles/docker_config/README.adoc

```diff
@@ -0,0 +1,9 @@
+== Role: Docker
```

Comment:
`== Role: Docker` should be `== Role: Configure Docker daemon`

Reply:
Fixed the heading